dot/sync/disjoint_block_set.go

```diff
@@ -125,21 +172,17 @@ func (s *disjointBlockSet) addHeader(header *types.Header) error {
 	defer s.Unlock()
 
 	hash := header.Hash()
-	b, has := s.blocks[hash]
-	if has {
+	if b, has := s.blocks[hash]; has {
```

Comment:
Let's use `ok` unless it has a special meaning, to fit with Go's standard lib 'convention'

```suggestion
	if b, ok := s.blocks[hash]; ok {
```

Reply:
i like has better as it's more descriptive, we're checking if the set has the block or not, but if you feel strongly i can change it